spec/lib/unit_tests_utils/consul_spec.rb

```diff
@@ -3,16 +3,18 @@
 
 describe UnitTestsUtils::Consul do
   let(:internal_consul_ip) { "1.2.3.4" }
-
-  before :each do
```

Comment:
You could do it like in InternalDNS where you left this one and just overwrote it in one case.

I do not have a preference between both solutions but I think you should pick just one unless there's a reason I am overseeing.